bootstrap

```diff
@@ -2355,11 +2355,11 @@ func_version ()
 # <https://github.com/gnulib-modules/bootstrap/issues>
 
 # Make sure we've evaluated scripts we depend on.
-test -z "$progpath" && . `echo "$0" |${SED-sed} 's|[^/]*$||'`/funclib.sh
-test extract-trace = "$progname" && . `echo "$0" |${SED-sed} 's|[^/]*$||'`/options-parser
+test -z "$progpath" && . "`echo "$0" |${SED-sed} 's|[^/]*$||'`funclib.sh"
```

Comment:
Seems like here is portability issue, there shouldn't be double quotes within ``"` ..here.. `"``, 
https://www.gnu.org/software/autoconf/manual/autoconf-2.64/html_node/Shellology.html
https://www.gnu.org/software/autoconf/manual/autoconf-2.64/html_node/Shell-Substitutions.html#Shell-Substitutions